src/components/Label/__stories__/examples/LabelExampleView/LabelExampleView.tsx

```diff
@@ -0,0 +1,185 @@
+                <Label style="default" size="xs">
+                    Default style
+                </Label>
+                <Label style="rounded" size="xs">
+                    Rounded style XS
+                </Label>
+                <Label style="rounded" size="s">
+                    Rounded style S
+                </Label>
+                <Label style="rounded" size="m">
+                    Rounded style M
+                </Label>
+            </DocsExample>
+        </Unstyled>
+    );
+}
+LabelExampleViewStyle.code = `
+<Label style="default" size="xs">Default style</Label>
+<Label style="rounded" size="xs">Rounded style XS</Label>
+<Label style="rounded" size="s">Rounded style S</Label>
+<Label style="rounded" size="m">Rounded style M</Label>
+`.trim();
+
+export function LabelExampleViewIcon() {
+    return (
+        <Unstyled>
+            <DocsExample>
+                <Label size="s" icon={<Icon size={16} data={GearIcon} />}>
+                    Default with Icon
+                </Label>
+                <Label theme="info" size="s" icon={<Icon size={16} data={GearIcon} />}>
+                    Info with icon
+                </Label>
+                <Label type="close" size="s" icon={<Icon size={16} data={GearIcon} />}>
+                    Close action and icon
+                </Label>
+                <Label size="m" icon={<Icon size={18} data={GearIcon} />}>
+                    M-size icon
+                </Label>
+            </DocsExample>
+        </Unstyled>
+    );
+}
+LabelExampleViewIcon.code = `
+<Label size="s" icon={<Icon size={16} data={GearIcon} />}>Default with icon</Label>
+<Label theme="info" size="s" icon={<Icon size={16} data={GearIcon} />}>Info with icon</Label>
+<Label type="close" size="s" icon={<Icon size={16} data={GearIcon} />}>Close action and icon</Label>
+<Label size="m" icon={<Icon data={GearIcon} />}>M-size icon</Label>
+`.trim();
+
+export function LabelExampleValue() {
+    return (
+        <Unstyled>
+            <DocsExample>
+                <Label size="xs" value="Value">
+                    Key
+                </Label>
+                <Label size="s" value="Value">
+                    Key
+                </Label>
+                <Label size="m" value="Value">
+                    Key
+                </Label>
+            </DocsExample>
+        </Unstyled>
+    );
+}
+LabelExampleValue.code = `
+<Label size="xs" value="Value">Key</Label>
+<Label size="s" value="Value">Key</Label>
+<Label size="m" value="Value">Key</Label>
+`.trim();
+
+export function LabelExampleState() {
+    return (
+        <Unstyled>
+            <DocsExample>
+                <Label size="s">Default</Label>
+                <Label size="s" disabled>
+                    Disabled
+                </Label>
+                <Label size="s" interactive>
+                    Interactive
+                </Label>
+            </DocsExample>
+        </Unstyled>
+    );
+}
+LabelExampleSize.code = `
```

Comment:
Shouldn't it be `LabelExampleState.code`?

Reply:
Oh, thank you, fixed